Content.Client/GameObjects/EntitySystems/ExamineSystem.cs

```diff
@@ -66,12 +70,44 @@ private bool HandleExamine(ICommonSession session, EntityCoordinates coords, Ent
             return true;
         }
 
+        public override void Update(float frameTime)
+        {
+            base.Update(frameTime);
+
+            if (_examinedEntity == null | _playerManager.LocalPlayer.ControlledEntity == null)
+            {
+                return;
+            }
+
+            // Can we even still see the item we examined?
+            if(!CanExamine(_playerManager.LocalPlayer.ControlledEntity, _examinedEntity))
+            {
+                CloseTooltip();
+                return;
+            }
+
+            var drawPos = _eyeManager.WorldToScreen(_examinedEntity.Transform.MapPosition.Position) + _examineOffset;
+
+            // If the draw point is outside the screen then just close the popup
+            // TODO
+
+            PopupContainer.SetPopupOrigin(_examineTooltipOpen, drawPos);
+        }
```

Comment:
Should this be FrameUpdate?

Reply:
I'm not sure, but what's the difference between FrameUpdate and Update? This code is way out of date but I am currently still using Update in the examine system

Reply:
FrameUpdate is called every render frame whereas Update is only called every tick (30 or 60 times a second normally).